main/ldb/APKBUILD

```diff
@@ -1,13 +1,13 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=ldb
-pkgver=1.3.6
+pkgver=1.4.3
 pkgrel=0
 pkgdesc="A schema-less, ldap like, API and database"
 url="http://ldb.samba.org/"
 arch="all"
 license="LGPL-3.0-or-later"
 makedepends="libtirpc-dev tevent-dev py2-tevent tdb-dev py2-tdb talloc-dev
-	python2-dev python3-dev popt-dev cmocka-dev"
+	python2-dev python3-dev popt-dev cmocka-dev lmdb-dev"
 subpackages="$pkgname-dev py2-$pkgname:_py2 py3-$pkgname:_py3 $pkgname-tools"
 source="https://www.samba.org/ftp/pub/ldb/$pkgname-$pkgver.tar.gz
 	disable-python-tests.patch
```

Comment:
#6011 suggests we don't need this patch at all.

Reply:
will check on that, thanks

Reply:
We need that patch, otherwise the tests won't finish on my machine.
See also the comment in the patch file: `Do not run python tests, they take eternity to finish.`

Reply:
I saw the comment. However, the [build](https://travis-ci.org/alpinelinux/aports/builds/477153122?utm_source=github_status&utm_medium=notification) for #6011 took less than four minutes to run.

What kind if machine are you using and how long did it take? It would be good to figure out the reason for the slow test you are seeing.